src/components/styles/PageSection.styled.ts

```diff
@@ -17,28 +17,28 @@ export const ImageContainer = styled.div<{ image: string }>`
   ${(props) => (props.image === '' ? `background-image: url(${Null});` : `background-image: url(${props.image});`)}
 `
 
-export const TextContainer = styled.div`
+export const TextContainer = styled.div<{ events?: boolean }>`
   height: 300px;
-  width: 50%;
+  width: ${(props) => (props.events ? '80%' : '50%')};
   display: flex;
   flex-direction: column;
 `
 
-export const Title = styled.div`
+export const Title = styled.div<{ events?: boolean }>`
   height: 45%;
-  text-align: center;
-  color: #0cbcbc;
+  text-align: ${(props) => (props.events === true ? 'left' : 'center')};
+  color: ${(props) => (props.events === true ? 'white' : props.theme.palette.primary)};
```

Comment:
boolean comparisons unecessary
```suggestion
  color: ${(props) => (props.events ? 'white' : props.theme.palette.primary)};
```